packages/@aws-cdk/aws-vpclattice-alpha/lib/service-network.ts

```diff
@@ -0,0 +1,150 @@
+import { CfnServiceNetwork } from 'aws-cdk-lib/aws-vpclattice';
+import { Stack, ArnFormat } from 'aws-cdk-lib/core';
+import { Construct } from 'constructs';
+import { IService } from './service';
+import { ServiceNetworkBase, IServiceNetwork } from './service-network-base';
+
+/**
+ * Authentication types for VPC Lattice service networks
+ */
+export enum ServiceNetworkAuthType {
+  /**
+   * No authentication
+   */
+  NONE = 'NONE',
+
+  /**
+   * AWS IAM authentication
+   */
+  AWS_IAM = 'AWS_IAM',
+}
+
+/**
+ * Properties for creating a VPC Lattice service network
+ */
+export interface ServiceNetworkProps {
+  /**
+   * The name of the service network
+   */
+  readonly serviceNetworkName: string;
+
+  /**
+   * The authentication type for the service network
+   *
+   * @default ServiceNetworkAuthType.NONE
+   */
+  readonly authType?: ServiceNetworkAuthType;
+
+  /**
+   * Whether to enable sharing of the service network
+   *
+   * @default false
+   */
+  readonly enableSharing?: boolean;
+
+  /**
+   * Tags for the service network
+   *
+   * @default - No tags
+   */
+  readonly tags?: string;
```

Comment:
```suggestion
  /**
   * Tags for the service network
   *
   * @default - No tags
   */
  readonly tags?: { [key: string]: string };
```

---

Required as per https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#tags
But I don't think we actually do this in a lot of places. @rix0rrr might have a better idea what the actual latest recommendation is here.